vmdb/app/models/ems_refresh/refreshers/openshift_refresher.rb

```diff
@@ -0,0 +1,14 @@
+module EmsRefresh
+  module Refreshers
+   class OpenshiftRefresher < BaseRefresher
```

Comment:
> Should this be a subclass of KubernetesRefresher since the parser is also a subclass?

I don't think so, because the parse_inventory method will look different because of another call to connect as discussed below wrt to amazon use case.